--- airtraf-1.1/src/sniffd/server.c.orig	2003-01-22 04:35:01.000000000 +0100
+++ airtraf-1.1/src/sniffd/server.c	2005-05-02 18:13:06.000000000 +0200
@@ -57,7 +57,7 @@
 
 int process_command(int, char *);
 int issue_get_cmd(int, char *);
-int send_bss_info(int);
+ssize_t send_bss_info(int);
 int send_ids_info(int);
 
 /*=============================================================*/
@@ -143,7 +143,7 @@
       if (DEBUG) fprintf(stderr,"Error transmitting data\n");
     }
   } while (write_len == 0);
-  if (DEBUG) fprintf(stderr,"Data: %d bytes sent\n",length);
+  if (DEBUG) fprintf(stderr,"Data: %d bytes sent\n",(int)length);
 }
 
 /**
